### ISSUE TITLE
Should add server g1 and g2

### ISSUE THREAD

**Issue by HackedGameMaker** [DxCx/plugin.video.9anime#130]
I've put a request in to add some things I'm not sure if you've seen them yet, one of those requests was to add servers g1 and g2 because they added them to their website.

**Comment by ThienJP** [comment]
here's an example link: https://9anime.to/watch/fullmetal-alchemist-brotherhood.0r7/8y8r1q

**Comment by miasmaejuices** [comment]
@DxCx - I've posted two pull requests to add the G1 and G2 servers.  They're a little hacky, as I don't understand the whole plugin, but I've tested them on a clean install of 18.0 ALPHA1-x64 (latest github nightly) and they work fine.

To get it to work on 18.0 ALPHA1, I had to install the latest script.module.urlresolver (4.0.08) from https://github.com/kodil/kodil.

Hope that saves you some time!
